### PR TITLE
Create browser test to ensure profile stability

### LINKED ISSUE #6069: Errors deserializing pac4j profiles
Seattle has seen warnings like the following in their prod logs:

```
[warn] o.p.c.u.s.JavaSerializer - cannot Java deserialize object
java.io.InvalidClassException: auth.CiviFormProfileData; local class incompatible: stream classdesc serialVersionUID = -4436848493580247884, local class serialVersionUID = 3142603030317816700
```

This means that users with existing profiles are not being recognized as existing users by a newer release of the server. This results in loss of state, including potentially applications in progress.

This warning has been logged 76 times in the 13 days since the last release.

The [changelog for v1.43.0](https://github.com/civiform/civiform/compare/v1.42.0...v1.43.0) contains a single PR that affects that class: [Refactor Account to AccountModel](https://github.com/civiform/civiform/pull/5966). This seems like a benign change that should not affect serialization, but it is possible that it does. See guidance from Oracle here: [Serializable (Java Platform SE 7)](https://docs.oracle.com/javase/7/docs/api/java/io/Serializable.html).

It is true that the class does not define a `serialVersionUID` field, which is an oversight. I've posted a question on StackOverflow for the `pac4j` team: [Errors deserializing pac4j profiles](https://stackoverflow.com/questions/77574171/errors-deserializing-pac4j-profiles).

### PR DESCRIPTION
**Pull request by yotommy**
### Description

Create a browser test to ensure profile stability.

This test validates that a previously-serialized profile can be properly deserialized by the current version of the server software. Since it is a browser test, it tests the system end-to-end, including filters, etc.

#### General

Read the full guidelines for PRs [here](https://github.com/civiform/civiform/wiki/Technical-contribution-guide#creating-a-pull-request)

- [x] Added the correct label: < feature | enhancement | bug | dependencies | infrastructure | ignore-for-release | database >
- [x] Assigned to a specific person, `civiform/developers`, or a [more specific round-robin list](https://github.com/civiform/civiform/wiki/Technical-contribution-guide#adding-reviewers).
- [x] Created unit and/or browser tests which fail without the change (if possible)

Fixes #6069

### CODE REVIEW

File: browser-test/src/profile_stability.test.ts [hunk 1]
```diff
@@ -0,0 +1,30 @@
+import {createTestContext} from './support'
+import {BASE_URL, FROZEN_PLAY_SESSION_COOKIE_VALUE} from './support/config'
+
+describe('developer tools', () => {
```
Comment (bion) [inline browser-test/src/profile_stability.test.ts:4]: **nitpick:** the test is focused on profiles/sessions, recommend:
```suggestion
describe('user HTTP sessions', () => {
```

File: browser-test/src/profile_stability.test.ts [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+  const ctx = createTestContext()
+
+  // This test ensures that an existing user profile, stored in a browser cookie,
+  // is recognized and properly deserialized by the server.
+  //
+  // This guards against changes that unexpectedly affect serialization.
+  it('recognizes the profile from a frozen cookie', async () => {
+    const hostname = new URL(BASE_URL).hostname
```
Comment (bion) [inline browser-test/src/profile_stability.test.ts:12]: **nitpick:** slightly more concise
```suggestion
    const { hostname } = new URL(BASE_URL)
```

File: browser-test/src/profile_stability.test.ts [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+    const frozenCookie = {
+      name: 'PLAY_SESSION',
+      value: FROZEN_PLAY_SESSION_COOKIE_VALUE,
+      domain: hostname,
+      expires: 2147483647,
```
Comment (bion) [inline browser-test/src/profile_stability.test.ts:17]: **thought (non-blocking):** maybe include a comment stating that this timestamp is in 2038, just to reduce cognitive load for someone reading this and trying to debug it

File: browser-test/src/profile_stability.test.ts [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+    }
+
+    await ctx.page.context().addCookies([frozenCookie])
+    await ctx.page.goto(BASE_URL + '/dev/profile')
+    const content = await ctx.page.content()
+
+    // The field order is non-deterministic, so just check some key fields.
+    expect(content).toContain('"id" : "1"')
+    expect(content).toContain('"roles" : [ "ROLE_APPLICANT" ]')
+    expect(content).toContain('"clientName" : "GuestClient"')
```
Comment (bion) [inline browser-test/src/profile_stability.test.ts:28]: **suggestion (if-minor, non-blocking):** could you parse the content JSON instead for cleaner looking asssertions?

Comment (yotommy) [reply browser-test/src/profile_stability.test.ts:33]: Done

**Pull request merged**